src/components/List.jsx

```diff
@@ -104,17 +116,23 @@ export default function List() {
               })
               .map((item, index) => {
                 return (
-                  <li key={index}>
-                    <input
-                      aria-label="checkbox for purchased item"
-                      id={item.data().id}
-                      type="checkbox"
-                      onChange={(e) => handleCheckBox(e, item)}
-                      checked={wasPurchasedWithin24Hours(item)}
-                      disabled={wasPurchasedWithin24Hours(item)}
-                    />
-                    {item.data().item}
-                  </li>
+                  <>
+                    <li key={index}>
+                      <input
+                        aria-label="checkbox for purchased item"
+                        id={item.data().id}
+                        type="checkbox"
+                        onChange={(e) => handleCheckBox(e, item)}
+                        checked={wasPurchasedWithin24Hours(item)}
+                        disabled={wasPurchasedWithin24Hours(item)}
+                      />
+                      {item.data().item}
+
+                      <button onClick={() => deleteHandler(item)}>
```

Comment:
nit: consistency of naming  - `deleteHandler` vs `handleCheckBox`

Reply:
Good call out! 

@simonkaine what are your thoughts on renaming one? I.e. should we have `deleteHandler` and `checkboxHandler` or `handleDelete` and `handleCheckbox`?

Reply:
Oh, nice call! I like the first of the two options here;`deleteHandler` and `checkboxHandler`. I think that seems to be more consistent in proper naming conventions for event handlers. I can go ahead and push these changes if we're all in agreement?

Reply:
@simonkaine that sounds good to me! Thanks! 😄

Reply:
Done. This change can be found in the commit; "Changed checkbox event handler name."